packs/feats/fey-ascension.json

```diff
@@ -28,7 +28,70 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Impossible Lands"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.proficiencies.spellcasting.rank",
+                "predicate": [
+                    {
+                        "or": [
+                            "fey-influence:anteater",
+                            "fey-influence:gremlin"
+                        ]
+                    }
+                ],
+                "value": 2
+            },
+            {
+                "category": "unarmed",
+                "damage": {
+                    "base": {
+                        "damageType": "slashing",
+                        "dice": 1,
+                        "die": "d6"
+                    }
+                },
+                "group": "dart",
+                "img": "icons/magic/nature/leaf-oak-glow-green.webp",
+                "key": "Strike",
+                "label": "PF2E.BattleForm.Attack.Leaves",
+                "predicate": [
+                    "fey-influence:dryad"
+                ],
+                "range": {
+                    "increment": 20
```

Comment:
No range was listed, so I went with dart range as it seemed to be similar to other unarmed ranged Strikes.

Reply:
Add a note to the description like we do when we use a DC from the chart